Code-the-Dream-School/dd-prac-team1-front
# Dev/responsive pages part2 (PRA-188)

## One Line Description
responsive edit recipe and responsive search with AI page
## Requirements

_What this code should do_

## Notes

_Any additional info_

## Test Steps
check that "find with AI page" and "edit recipe page"  are responsive
_Instructions on how to test the changes_

## UI/UX Outcome

_Attach a screenshot here_

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
